### DISCUSSION TITLE
OSQuery data in Kibana- how to view?

### DISCUSSION THREAD

**Discussion by adamphetamine** [Security-Onion-Solutions/securityonion#9693] [Unsupported Versions] [answered]
Hi All,
I've probably missed something simple so please be gentle!

I can see that osquery is sending data to Kibana, because I can click on 'Event Category: Host' and a bunch of stuff appears in 
'Security onion- Dataset' - and this includes the query packs I've added and the custom queries I have running in Fleet.

How do I browse or access this data?

If I click on any of the pack names it opens a new window and the hyperlink goes to (roughly)

https://my.so-install.com/kibana/app/pack_mac-pack_firefox_addons

and gives the error 'application not found' 

Similarly, when using the panel 'Security Onion- Modules' and clicking on 'OSQuery' I get an error 'Add OSQuery Manager' but I had assumed this wouldn't work because of the SO customisation...

My SO instance is set up as a 'Manager Search' node

![Screenshot 2023-02-01 at 6 24 58 pm](https://user-images.githubusercontent.com/47110465/215978098-54b14ab3-1551-42b9-9934-760f239bc3d5.png)

**Comment by InfosecGoon** [comment, marked as answer]
Unfortunately, this is an issue with field formatters not working properly in Elastic 8.5 -- it will be fixed with the move to 8.6.1, as tracked in this issue: https://github.com/Security-Onion-Solutions/securityonion/issues/9594

I would suggest using Hunt for now to access those OSQuery results until 2.3.210 is available with the new Elastic.

**Comment by adamphetamine** [comment]
Thanks so much @InfosecGoon,
I thought I was going nuts.

Cheers
